### DISCUSSION TITLE
RDf / Semantic Web

### DISCUSSION THREAD

**Discussion by uahic** [apache/age#1706] [General]
Hi,

I would just like to briefly ask if there are any plans or already ongoing developments of adapters between semantic web formats such as RDF or Turtle to Apache AGE?  Currently I am using RDF4J + Halyard (an adapter for Apache HBase) to achieve this and use Postgre on the side for user management etc.

Alternatively, is there a way in Apache AGE to define "ontology"/graph schemas, validators and potential deduction?

Thank you !

**Comment by markgomer** [comment]
@jrgemignani @dehowef @rafsun42 or @eyab , do you have any updated information on this?

**Comment by mingfang** [comment]
KuzuDB has native support for RDF
https://docs.kuzudb.com/rdf-graphs/

**Reply by uahic** [reply]
The thing is that I dont want to revert back to small projects with a few maintainers but would love to see this particularly in a major DB like Postgre/Age where you benefit from all the other plugins as well. There are way too many small specialized databases (I've compiled a list for knowledge graphs - and it is very long) which will dissappear in  a few months-few years

**Reply by clxyder** [reply]
Seems like Kuzu removed support for RDF and are currently debating whether to bring it back https://github.com/kuzudb/kuzu/issues/4991
